### PR TITLE
Issue #1698: Preliminary UsageRecords Integration test

### PR DESCRIPTION
**Pull request by abdulazizali77**

### CODE REVIEW

**Comment by deruelle** [comment]
Thanks @abdulazizali77 ! @gvagenas @otsakir can you please review ?

**Comment by abdulazizali77** [comment]
@deruelle @otsakir Sorry this isnt absolutely done yet!

**Comment by otsakir** [comment]
@abdulazizali77, i ran the tests against restcomm_with_Data.script.  the following tests failed:

    UsageRecordsTest.getUsageRecordsDaily
    UsageRecordsTest.getUsageRecordsMonthly
    UsageRecordsTest.getUsageRecordsYearly
    UsageRecordsTest.getUsageRecordsAlltime

Is this expected since https://github.com/RestComm/Restcomm-Connect/pull/1696 is not yet merged ?

**Comment by abdulazizali77** [comment]
@otsakir yes, failing these is a good thing :D when the rest get merged, i will update these

File: restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/UsageRecordsTest.java [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+                .resolve("org.restcomm:restcomm-connect.application:war:" + version).withoutTransitivity()
+                .asSingle(WebArchive.class);
+        archive = archive.merge(restcommArchive);
+        archive.delete("/WEB-INF/sip.xml");
+        archive.delete("/WEB-INF/conf/restcomm.xml");
+        archive.delete("/WEB-INF/data/hsql/restcomm.script");
+        // archive.delete("/WEB-INF/data/hsql/restcomm.properties");
+        archive.addAsWebInfResource("sip.xml");
+        archive.addAsWebInfResource("restcomm.xml", "conf/restcomm.xml");
+        archive.addAsWebInfResource("restcomm_with_Data_UsageRecords.script", "data/hsql/restcomm.script");
```
Comment (abdulazizali77) [inline restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/UsageRecordsTest.java]: @otsakir i updated the commit, i remember adding the `restcomm_with_Data_UsageRecords.script` script, i guess i was wrong!
You are right about the params, but i did not intend to change/use `restcomm_with_Data.script`.

**Comment by otsakir** [comment]
@abdulazizali77 , your PR has been merged. Great work work there. 

Thanks for contributing! :+1:

**Comment by abdulazizali77** [comment]
thank you @otsakir !

**Pull request merged**